react/frontpage/EventsContent.jsx

```diff
@@ -0,0 +1,129 @@
+// EventsContent.jsx
+var React = require('react');
+
+var Modal = require('react-bootstrap').Modal;
+var Input = require('react-bootstrap').Input;
+var FormControls = require('react-bootstrap').FormControls;
+var Button = require('react-bootstrap').Button;
+if (typeof Button === 'undefined' || Button === null) {
+    console.log("issues!!\n");
+}
+
+var EventsContent = React.createClass({
+	render: function() {
+		return(
+			 <div className='wf-container'>
+			{ JSON.parse(this.props.data).events.map (function(el) {
+					return (
+						<div className="monthEvents" key={el['month']}>
+							<h1>{el['month']}</h1>
+							<div className="eventBlock">
+								{el['arr'].map( function(event){
+									return (
+										<Event event={event} key={event['id']}/>
+									);
+								})
+								}
+							</div>
+						</div>
+					);
+				})
+			}
+			</div>
+		);
+	}
+});
+
+var Event = React.createClass({
+	getInitialState: function(){
+		return{
+			hover: false,
+			showModal: false
+		};
+	},
+	mouseEnter: function(){
+		this.setState({hover: true});
+	},
+	mouseLeave: function(){
+		this.setState({hover: false});
+	},
+	showModal: function(){
+		this.setState({showModal: true})
+	},
+	hideModal: function(){
+		this.setState({showModal: false})
+	},
+	render: function(){
+		var hoverStyle = {display: "none"};
+		if(this.state.hover){
+			hoverStyle = {height: 200, width: 200, backgroundColor: "grey", opacity:"0.8", position: "absolute", bottom: 0, left: 0, right: 0, display: "inline"};
+		}
+		return(
+		<div>
+			<div className="event" onClick={this.showModal} onMouseEnter={this.mouseEnter} onMouseLeave={this.mouseLeave}>
+				<p className="bandName">{getBandName(this.props.event['summary'])}</p>
+				<img src="https://avatars3.githubusercontent.com/u/7256558?v=3&s=466" className="bandPic"/>
```

Comment:
Yes, according to the mockups. I think we're confirming that with Promotions soon too.